.golangci.yml

```diff
@@ -10,7 +10,7 @@ linters:
     - deadcode
     - depguard
     - dogsled
-    # - errcheck
+    - errcheck
```

Comment:
Do we really want that? This is part of https://github.com/cosmos/cosmos-sdk/issues/7258 which is not completely implemented through the whole codebase.

Reply:
I can remove this from the file entirely.